molecule/config.py

```diff
@@ -403,9 +401,8 @@ def _get_defaults(self):
                 'create_sequence': ['dependency', 'create', 'prepare'],
                 'destroy_sequence': ['dependency', 'cleanup', 'destroy'],
                 'test_sequence': [
-                    # dependency must be kept before lint to avoid errors
-                    'dependency',
                     'lint',
+                    'dependency',
```

Comment:
CI failure is because of this or?

> https://travis-ci.com/ansible/molecule/jobs/231441356#L2052

Why was this changed? Looks unrelated.

Reply:
Maybe a rebase problem?

Reply:
Good catch 👍 Must have been a sloppy rebase, fixed now!